cluster/gce/gci/master-helper.sh

```diff
@@ -36,15 +36,16 @@ function create-master-instance {
 
   write-master-env
   ensure-gci-metadata-files
-  create-master-instance-internal "${MASTER_NAME}" "${address}"
+  create-master-instance-internal "${master_name}" "${address}"
```

Comment:
Thanks @xichengliudui
But I doubt the correctness of this change because `${MASTER_NAME}` can come from the upstream.
- https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/config-default.sh#L111
- https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/config-test.sh#L111

/cc @fejta

Reply:
Yeah this doesn't look right

Reply:
Wait a moment